### PR TITLE
Remove unnecessary needle from readme

### PR DESCRIPTION
**Pull request by RothAndrew**
## what
Remove unnecessary needle from when we were trying to get markdown-lint inline ignores working

## why
It isn't looked at by Codacy

## references
n/a

### CODE REVIEW

**Review by Xerralt — APPROVED** [review]

**Comment by RothAndrew** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/saic-oss/terraform-aws-rke-rancher-master-cluster/pullRequest?prid=6339862&bid=20593846)

File: README.md [hunk 1]
```diff
@@ -181,11 +180,11 @@ This module will use SemVer, and will stay on v0.X for the foreseeable future
 
 | Name | Description |
 |------|-------------|
-| cluster\_kubeconfig | KUBECONFIG yaml file contents to connect to the cluster. DO NOT USE unless you have no other options. Users should use the KUBECONFIG that Rancher provides to them rather than this. |
+| cluster\_kubeconfig | [SENSITIVE] KUBECONFIG yaml file contents to connect to the cluster. DO NOT USE unless you have no other options. Users should use the KUBECONFIG that Rancher provides to them rather than this. |
```
Comment (RothAndrew) [inline README.md:183]: Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/saic-oss/terraform-aws-rke-rancher-master-cluster/pullRequest?prid=6339862)

File: README.md [hunk 2]
```diff
@@ -164,7 +163,7 @@ This module will use SemVer, and will stay on v0.X for the foreseeable future
 | hosted\_zone\_domain\_name | Domain name of the hosted zone to create records in | `string` | n/a | yes |
 | hosted\_zone\_id | ID of Route53 hosted zone to create records in | `string` | n/a | yes |
 | instance\_type | Instance type to use for the cluster nodes | `string` | n/a | yes |
-| kubernetes\_version | Kubernetes version to use. Must be supported by the version of the RKE provider you are using. See [https://github.com/rancher/terraform-provider-rke/releases](https://github.com/rancher/terraform-provider-rke/releases) | `string` | n/a | yes |
+| kubernetes\_version | Kubernetes version to use. Must be supported by the version of the RKE provider you are using. See https://github.com/rancher/terraform-provider-rke/releases | `string` | n/a | yes |
```
Comment (RothAndrew) [inline README.md:166]: Codacy found an issue: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/gh/saic-oss/terraform-aws-rke-rancher-master-cluster/pullRequest?prid=6339862)

File: README.md [hunk 3]
```diff
@@ -181,11 +180,11 @@ This module will use SemVer, and will stay on v0.X for the foreseeable future
 
 | Name | Description |
 |------|-------------|
-| cluster\_kubeconfig | KUBECONFIG yaml file contents to connect to the cluster. DO NOT USE unless you have no other options. Users should use the KUBECONFIG that Rancher provides to them rather than this. |
+| cluster\_kubeconfig | [SENSITIVE] KUBECONFIG yaml file contents to connect to the cluster. DO NOT USE unless you have no other options. Users should use the KUBECONFIG that Rancher provides to them rather than this. |
 | rancher\_admin\_password | Password for Rancher 'admin' user |
 | rancher\_admin\_token | API Token for Rancher 'admin' user |
 | rancher\_endpoint | Endpoint of Rancher Server |
-| ssh\_private\_key | Cluster nodes' private SSH key |
+| ssh\_private\_key | [SENSITIVE] Cluster nodes' private SSH key |
```
Comment (RothAndrew) [inline README.md:187]: Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/saic-oss/terraform-aws-rke-rancher-master-cluster/pullRequest?prid=6339862)

**Comment by RothAndrew** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/saic-oss/terraform-aws-rke-rancher-master-cluster/pullRequest?prid=6339862&bid=20593846)

File: README.md (hunk 1 above)
Comment (RothAndrew) [inline README.md:183]: Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/saic-oss/terraform-aws-rke-rancher-master-cluster/pullRequest?prid=6339862)

File: README.md (hunk 2 above)
Comment (RothAndrew) [inline README.md:166]: Codacy found an issue: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/gh/saic-oss/terraform-aws-rke-rancher-master-cluster/pullRequest?prid=6339862)

File: README.md (hunk 3 above)
Comment (RothAndrew) [inline README.md:187]: Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/saic-oss/terraform-aws-rke-rancher-master-cluster/pullRequest?prid=6339862)

**Review by ac-miller — APPROVED** [review]

**Pull request merged**